examples/library/test_struct_size/main.c

```diff
@@ -26,7 +26,6 @@
 // SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
```

Comment:
Is this test superseded by test_packet?

Reply:
lol - whoops. forgot this exists.

Reply:
in my defense, what a non-descript name?

Reply:
I think we can nuke this.

Reply:
Should I get rid of it then before we merge?

Reply:
separate PR I think